MLS.Agent/CommandLine/CommandLineParser.cs

```diff
@@ -181,6 +181,11 @@ RootCommand StartInTryMode()
                                       "Enable verbose logging to the console",
                                       new Argument<bool>()));
 
+                command.AddOption(new Option(
+                                        "--port",
+                                        "Specify the port for dotnet try to listen on",
+                                        new Argument<int>()));
```

Comment:
negative numbers are not valid

Reply:
Type should probably be `ushort`, since that matches the sign/max value requirements.